reference/touchsensor.md

```diff
@@ -125,6 +125,9 @@ robot) makes the actual contact with the floor. Therefore, it is necessary that
 the sensor's `boundingObject` extend below any other `boundingObject` of the
 robot (e.g., foot, ankle, etc.).
 
+If the [TouchSensor](#touchsensor) node doesn't have any dynamic ancestor [Solid](#solid) node, i.e. any node whose `physics` field is set, it will be attached to the static environment.
```

Comment:
Not sure it is interesting or understandable for the user to know the details about how it is fixed to the static environment, but we can add it.